dask_jobqueue/sge.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import os
+import sys
+
+from .core import JobQueueCluster, docstrings
+
+logger = logging.getLogger(__name__)
+
+dirname = os.path.dirname(sys.executable)
+
+
+@docstrings.with_indent(4)
+class SGECluster(JobQueueCluster):
+    """ Launch Dask on a SGE cluster
+
+    Parameters
+    ----------
+    queue : str
+        Destination queue for each worker job. Passed to `#$ -q` option.
+    project : str
+        Accounting string associated with each worker job. Passed to
+        `#$ -A` option.
+    resource_spec : str
+        Request resources and specify job placement. Passed to `#$ -l`
+        option.
+    walltime : str
+        Walltime for each worker job.
+    interface : str
```

Comment:
You still have the interface keyword in this docstring, once remove I think we're done :)

Reply:
Arrggh good point, I just pushed the fix.